feed_amalgamator/feed_amalgamator/feed.py

```diff
@@ -97,7 +97,7 @@ def add_server():
                 try:
                     db.execute(
                         "INSERT INTO user_server (user_id, server, token) VALUES (?, ?, ?)",
-                        (user_id, domain, auth_token),
+                        (user_id, domain, users_access_token),
```

Comment:
Is the `users_access_token` consistent across multiple Mastodon sessions? (The `auth_token` remains constant)

Reply:
Yup it is persistent when logging out and back in as well as restarting the server
. I think the previous problem is that the auth_token is only one time use when calling bot_m.log_in() to get the users_access_token. So when I tried to logout and log back in it said that the token was invalid.